cumulusci/tasks/apex/testrunner.py

```diff
@@ -115,23 +112,24 @@ class RunApexTests(BaseSalesforceApiTask):
         },
         "poll_interval": {
             "description": (
-                "Seconds to wait between polling for Apex test "
-                + "results.  Defaults to 3"
+                "Seconds to wait between polling for Apex test " + "results."
             )
         },
-        "retries": {"description": "Number of retries (default=10)"},
-        "retry_interval": {
-            "description": "Number of seconds to wait before the next retry (default=5),"
-        },
-        "retry_interval_add": {
-            "description": "Number of seconds to add before each retry (default=5),"
-        },
         "junit_output": {
             "description": "File name for JUnit output.  Defaults to test_results.xml"
         },
         "json_output": {
             "description": "File name for json output.  Defaults to test_results.json"
         },
+        "retry_errors": {
```

Comment:
Should this be `retry_failures`?

Reply:
@davidmreed

Reply:
@davisagli  I'm probably missing something obvious - what's the impact of the terminology?

Reply:
@davidmreed my thinking is just that the key we're looking for in the test results is `Fail`, not `Error`, so calling this errors feels like making up our own terminology.

Reply:
Oh, I see. `failures` probably makes more sense. I will update it.